client/blocks/quests/index.js

```diff
@@ -48,7 +48,7 @@ const Objective = connectObjective(({
 	}
 
 	{objective.completed && <Completed>
-		{new OdreianDate(objective.completedDate).format`${'llll'}`}
+		{new OdreianDate(objective.completedDate).format`${'h'}:${'mm'}${'a'}, ${'dddd'}, ${'Do'} of ${'MM'}, ${'YY'}`}
```

Comment:
Note: this is hardcoded for now until I tackle https://github.com/rowanmanning/dream-date/issues/9